raiden/raiden_service.py

```diff
@@ -545,7 +548,7 @@ def start(self):
 
         self._initialize_payment_statuses(chain_state)
         self._initialize_transactions_queues(chain_state)
-        self._initialize_messages_queues(chain_state)
+        # self._initialize_messages_queues(chain_state)
```

Comment:
This function is executed later. Remove the comment

Reply:
Done